### ISSUE TITLE
Locus package def., questions

### ISSUE THREAD

**Issue by tortuosit** [Falcosc/locus-addon-tasker#15]
Hi, two ideas as for the Locus version picker.

a) Please allow to use a variable for the Locus package name. I am holding it in a global var and could use it

b) Wouldn't "Locus package" be better placed in any of your commands?

Questions. Is the Locus Version command a kind of global toggle, i. e. your commands will use the chosen Locus version in the future? Until we use the version command again?

Speed is in mph, right? ie, km/h=speed*3.6...?

Thanks

**Comment by Falcosc** [comment]
You pick Version only once and it will stay until reboot. You don't need global variables. Just select it once and keep it as separate task.

**Comment by tortuosit** [comment]
Thing is: I automate both versions and do things depending on the version. But np, I can use multiple conditional commands of your plugin.

**Comment by Falcosc** [comment]
I thought about a version pick-list on the app main screen to make it more clear that this selection is permanent and to make it more transparent which version is selected right now.

**Comment by tortuosit** [comment]
Since we are in the tasker realm, I'd prefer a pick list in any of your commands. So you can pick the version inside the Tasker dialog (eg "Request sensors and stats"). Optionally. And with the possibility to use a variable for the package name.
IMO, this would totally be the Tasker way.

In the app main screen a preferred version, to be optionally overridden inside Tasker.

**Comment by Falcosc** [comment]
That will be worse then extending the select version command because it will not be readable if every configuration is extended by an additional package line.

Don't forget that there is only a single line of text available to describe the task configuration.

By the way, how do you control both versions simultaneously? Which use cases do need 2 locus map applications?

**Comment by tortuosit** [comment]
I am doing manual hiking recording and automated "leave home" recording. They aren't running at the same time.
LM3 for auto recordings, LM4 everything else. Why? I dislike syncing my home position, home behaviour almost, to a public server. As you know LM4 currently can only sync all or nothing.

But tbh, I don't want 2 instances any more. So I just use LM4 and don't do syncing.

**Comment by tortuosit** [comment]
So I admit the version picker currently is not important for me any more. 😑

**Comment by Falcosc** [comment]
@tortuosit I did add a version select option for auto-detection and 6 other things
https://play.google.com/apps/testing/falcosc.locus.addon.tasker

You may need to wait some hours until the version from today is available.
